### DISCUSSION TITLE
[1.0-BETA] 2021-08-23

### DISCUSSION THREAD

**Discussion by darthcloud** [darthcloud/BlueRetro#64] [General]
# Wireless
* Add support for BLE generic HID gamepad, mouse & keyboard
* Add support for Xbox Series X|S controller
* Add workaround for PowerA switch controller stopping report after inquiry stop

# Adapter
* Improve Generic HID gamepad default mapping
* Introduce default mapping quirks for inverted and rotated face buttons (8bitdo, PowerA GC)

# Config
* Add version string attribute (Displayed in OTA page logs)

<hr /><em>This discussion was created from the release <a href='https://github.com/darthcloud/BlueRetro/releases/tag/v1.0-beta'>[1.0-BETA] 2021-08-23</a>.</em>

**Comment by darthcloud** [comment]
# Known issues
* Rumble is broken and will crash the adapter
* CDi, PSX & Saturn keyboard are broken and will crash adapter

**Comment by vaderag** [comment]
Just wondering if the known issues apply to all firmwares? E.g. if I use the GC only one? Thanks!

**Reply by darthcloud** [reply]
Yes it affect all FW, they are all the same except system type init.
